modules/ROOT/pages/worker-monitoring.adoc

```diff
@@ -1,102 +1,93 @@
-= Worker Monitoring
+= Application Monitoring and Automatic Restarts
 ifndef::env-site,env-github[]
 include::_attributes.adoc[]
 endif::[]
 :keywords: cloudhub, monitoring, worker, restarts
 
 image:logo-cloud-active.png[link="/runtime-manager/deployment-strategies", title="CloudHub"]
-image:logo-hybrid-disabled.png[link="/runtime-manager/deployment-strategies", title="Hybrid Deployment"]
-image:logo-server-disabled.png[link="/runtime-manager/deployment-strategies", title="Anypoint Platform Private Cloud Edition"]
-image:logo-pcf-disabled.png[link="/runtime-manager/deployment-strategies", title="Pivotal Cloud Foundry"]
+image:logo-hybrid-disabled.png[link="/runtime-manager/deployment-strategies", title="Hybrid"]
+image:logo-server-disabled.png[link="/runtime-manager/deployment-strategies", title="Anypoint Platform PCE"]
+image:logo-rtf-disabled.png[link="/runtime-fabric/deploy-to-runtime-fabric", title="Runtime Fabric"]
 
-[NOTE]
-Worker Monitoring is a feature that is only available for applications deployed to CloudHub workers. It is not available for applications that you deploy using the Runtime Manager to local servers. See xref:deployment-strategies.adoc[Deployment Strategies] and xref:monitoring.adoc[Monitoring Applications] for more details.
 
-CloudHub supports worker monitoring and automatic restarts for unresponsive applications. Monitoring allows you the option to receive notifications and diagnostic information if your application becomes unresponsive. Optionally, enable automatic restarts so that your application can recover automatically without your intervention. CloudHub gives you the flexibility to conduct monitoring with or without enabling automatic restarts.
+CloudHub monitors all applications and enables you to receive notifications and diagnostic information if your application becomes unresponsive.
 
-Monitoring is enabled by default on all applications deployed on CloudHub. Applications already deployed before the monitoring feature was introduced need to be redeployed for monitoring to take effect. 
+You can enable automatic restarts so that your application can recover automatically without your intervention.
 
-You can activate the automatic restart feature on a per-application basis using the Runtime Manager console when you deploy a new application or redeploy an existing application.
 
-== Prerequisites
+== How Application Monitoring Works
 
-This document assumes that you are familiar with xref:deploying-to-cloudhub.adoc[Deploying to CloudHub] and that you are deploying to runtime 3.4.n, 3.5.n, or newer.
+Monitoring works differently depending on your Mule version.
 
-== Enabling Automatic Restarts
+* Mule 4.x
++
+Every 30 seconds, CloudHub checks the PID of the Mule process.
++
+If CloudHub detects the application process is not running three times in a row, the logs print a warning.
```

Comment:
```suggestion
If CloudHub detects that the application process is not running three times in a row, the logs print a warning.
```
Should we clarify "three times in a row"? What time span are we looking at here? or should we say "After an application stops running three times, CloudHub sends a warning to the application logs."